src/docs.js

```diff
@@ -4,44 +4,76 @@ let assert = require('assert');
 let fs = require('mz/fs');
 let path = require('path');
 let recursiveReadSync = require('recursive-readdir-sync');
+let zlib = require('zlib');
+let rootdir = require('app-root-dir');
 
 async function documenter(options) {
-
   options = _.defaults({}, options, {
+    tier: null,
+    menuIndex: null,
     schemas: [],
-    docsFolder: null,
+    docsFolder: rootdir.get() + '/docs',
+    references: [],
   });
+
   assert(options.schemas, 'options.schemas must be given');
   assert(options.schemas instanceof Array, 'options.schemas must be an array');
 
-  let schemas = options.schemas;
+  assert(options.tier, 'options.tier must be given');
+  assert(['core', 'platform'].indexOf(options.tier) !== -1, 'options.tier is either core or platform');
+  assert(options.menuIndex, 'options.menuIndex must be given');
+
   let tarball = tar.pack();
+  let metadata = {version: 1, tier: options.tier, menuIndex: options.menuIndex};
+
+  if (metadata) {
+    let data = JSON.stringify(metadata, null, 2);
+    tarball.entry({name: 'metadata.json'}, data);
+  }
 
-  // add schemas to tarball
+  let schemas = options.schemas;
   schemas.forEach(schema => {
```

Comment:
Yes i was planning to fix it in the next commit ( publish tarball to s3). Sorry for the delay.